lib/charms/loki_k8s/v1/loki_push_api.py

```diff
@@ -2313,6 +2423,265 @@ def _containers(self) -> Dict[str, Container]:
         return {cont: self._charm.unit.get_container(cont) for cont in self._logs_scheme.keys()}
 
 
+class _LogForwarderHelpers:
+    @staticmethod
+    def check_juju_version() -> bool:
+        """Make sure the Juju version supports Log Forwarding."""
+        juju_version = JujuVersion.from_environ()
+        if not juju_version > JujuVersion(version=str("3.3")):
+            msg = f"Juju version {juju_version} does not support Pebble log forwarding. Juju >= 3.4 is needed."
+            logger.warn(msg)
```

Comment:
`warn` is deprecated. Use `logger.warning` instead.
```suggestion
            logger.warning(msg)
```

Reply:
Done!